Userland/Shell/Builtin.cpp

```diff
@@ -1921,7 +1923,7 @@ ErrorOr<int> Shell::builtin_run_with_env(Main::Arguments arguments)
 
 bool Shell::has_builtin(StringView name) const
 {
-    if (name == ":"sv)
+    if (name == ":"sv || name == "."sv)
```

Comment:
```suggestion
    if (name.is_one_of(":"sv, "."sv))
```

Reply:
Actually, please limit this one to posix mode.
```suggestion
    if (name == ":"sv || (m_posix_mode && name == "."sv))
```